packages/google-cloud-access-context-manager/noxfile.py

```diff
@@ -458,3 +468,70 @@ def prerelease_deps(session, protobuf_implementation):
             "PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION": protobuf_implementation,
         },
     )
+
+
+@nox.session(python="3.13")
+@nox.parametrize(
+    "protobuf_implementation",
+    ["python", "upb"],
+)
+def core_deps_from_source(session, protobuf_implementation):
```

Comment:
Maybe make it clear it's the local source, as opposed to checking it out from somewhere? 

```suggestion
def core_deps_from_local_source(session, protobuf_implementation):
```

Reply:
`core_deps_from_source` seems correct to me as we are installing packages from source which is not necessarily local. For example, currently we have the following installations which are remote. I also filed TODO bugs to add installing `protobuf` and `grpc` from source, which will also be remote.
```
        "proto-plus @ git+https://github.com/googleapis/proto-plus-python.git",
        "google-api-core @ git+https://github.com/googleapis/python-api-core.git",
        "google-auth @ git+https://github.com/googleapis/google-auth-library-python.git",
```

See https://github.com/googleapis/gapic-generator-python/issues/2357 and https://github.com/googleapis/gapic-generator-python/issues/2358 for the bugs to follow up on this.

Reply:
I see your point. I was focusing on the relative dirs you have in there......but OK, leaving the name as is does sound better. It's implied (maybe we should make it explicit) that for anything that comes from this repo, `google-cloud-python`, we'll incorporate local changes.